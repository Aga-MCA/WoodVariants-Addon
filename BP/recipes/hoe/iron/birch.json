{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "iron_birch_hoe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:birch_stick"
      },
      "P": {
        "item": "minecraft:iron_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:iron_birch_hoe"
    }
  }
}