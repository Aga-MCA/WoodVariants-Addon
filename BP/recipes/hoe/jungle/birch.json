{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "jungle_birch_hoe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:birch_stick"
      },
      "P": {
        "item": "aga_wood_variants:jungle_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:jungle_birch_hoe"
    }
  }
}