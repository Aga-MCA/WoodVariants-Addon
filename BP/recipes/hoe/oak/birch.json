{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "oak_birch_hoe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:birch_stick"
      },
      "P": {
        "item": "aga_wood_variants:oak_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:oak_birch_hoe"
    }
  }
}