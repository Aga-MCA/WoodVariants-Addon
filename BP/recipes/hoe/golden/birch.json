{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_birch_hoe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:birch_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_birch_hoe"
    }
  }
}